sentry_sdk/profiler/continuous_profiler.py

```diff
@@ -51,9 +52,10 @@
 
 
 try:
-    from gevent.monkey import get_original  # type: ignore
-    from gevent.threadpool import ThreadPool  # type: ignore
+    from gevent.monkey import get_original
+    from gevent.threadpool import ThreadPool as _ThreadPool
 
+    ThreadPool = _ThreadPool  # type: Optional[Type[_ThreadPool]]
```

Comment:
Why is this necessary?

Reply:
If we don't do this, then mypy will complain about redefinition of `ThreadPool` to `None` in the `except` clause. So the type and the variable needs a separate name, so I can use the type separately from the runtime variable.